src/main/java/com/scriptbasic/interfaces/ScriptBasicKeyWords.java

```diff
@@ -5,11 +5,19 @@
  * date June 26, 2012
  */
 public interface ScriptBasicKeyWords {
-    String[] BASIC_KEYWORDS = new String[]{"for", "end", "next", "let", "if",
-            "endif", "then", "while", "wend", "else", "elseif", "repeat",
+	public static final String KEYWORD_CASE = "case";
```

Comment:
Why do you feel that you need these constants?

If you seriously think that there is a need for these keywords then please create a separate pull request that creates a symbolic name for each keyword. This version is not consistent.